source/server/config/http/ip_tagging.cc

```diff
@@ -0,0 +1,35 @@
+#include "server/config/http/ip_tagging.h"
+
+#include <string>
+
+#include "common/http/filter/ip_tagging_filter.h"
+#include "common/json/config_schemas.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+
+HttpFilterFactoryCb IpTaggingFilterConfig::tryCreateFilterFactory(HttpFilterType type,
```

Comment:
can you refactor this to use a new method for filter registration, see https://github.com/lyft/envoy/commit/3a466c3506a97df38a9efb15dff85e2a7e75b556#diff-12a4439b6c76df424840b91ab1cf3789R14